docs/self-hosted/kubernetes-embedded.md

```diff
@@ -37,7 +37,12 @@ in an existing Kubernetes cluster using Helm follow [this guide](./kubernetes-he
   * s3.kurl.sh (required to install the kots CLI)
   * amazonaws.com (required to install the kots CLI)
   * Please also consult the list in the [Replicated documentation](https://docs.replicated.com/enterprise/installing-general-requirements#firewall-openings-for-online-installations).
-
+  * To mirror Composer packages in Private Packagist, the following domains must also be accessible:
+     - packagist.org (required to fetch package metadata)
+     - *.packagist.org (required to fetch package metadata)
+     - github.com (required to download ZIP files for mirrored packages)
+     - api.github.com (required to download ZIP files for mirrored packages)
```

Comment:
If we mention GitHub, shouldn't we also include Bitbucket + GitLab, or even more in general whatever VCS they use?

Reply:
99% of all packages on packagist.org are hosted on GitHub. So I wouldn't worry about that too much

Reply:
OK, makes sense.